### ISSUE TITLE
[Enhancement]: Bulk Chapters Lookup

### ISSUE THREAD

**Issue by raviwarrier** [advplyr/audiobookshelf#1580]
### Describe the feature/enhancement

Right now, I face two issues which this feature should solve:

1. I generally upload a few books together at a time. I have to individually look up (match/quick match) books [see #1490 for another similar feature request] and then click on chapters tab to lookup chapters. 
2. I have a lot of books that I uploaded before the chapter lookup feature was implemented and all those books need to be individually updated for chapters. 

It would be nice if there was a way to select books and hit "update chapters" button to do this automatically. Coupled with #1490 for new books, this would complete the requirement for book and chapter look up.

**Comment by advplyr** [comment]
For this I think we would have to ensure the duration of the audiobook chapters from Audible matches almost exactly what the duration is of your audiobook. I think a lot of people have the Audible intro removed from the audio file.

**Comment by Dr-Blank** [comment]
Could abshelf do the same?

If there is not an exact match, try to match within 6 seconds of the runtime, because it is usually that long, right?

6 seconds will be sort of an acceptable error.

**Comment by advplyr** [comment]
I'm not sure. It could be variable how much they trim then we would have to adjust the timestamps.

**Comment by Dr-Blank** [comment]
of course, the timestamps are variable, hence it would not be fixed to 6 seconds, just adjust the time stamp with whatever is the difference, as long as it is within a set time (for example 6 seconds). 

And a chapter difference by a few seconds is definitely better than no chapters at all.

**Comment by raviwarrier** [comment]
I just left a new enhancement request #1811 and I wanted to re-iterate how much I feel the need for this feature. I just updated 29 books (quick matched), but now I have to go individually to each book and look up chapters. Only if there was an option to automatically update chapters along with updating matches/info (either individually or in bulk).

I understand that sometimes durations don't match, but for that here could be some possible solutions:
1. In settings, along with "automatically update chapters" toggle, you could also have "accept all", "reject all" to allow the admin/user to choose to update even if the chapters don't completely align.
2. For bulk updates, you could pop a new tab in settings "Resolve discrepancies" where you show the list of the books with discrepancies and a button to accept/reject.

#2 can be enhanced a bit:
a. you could include type of discrepancy - book info not matching, chapter not matching, etc.
b. you could have a (i) button next to each book to show the details and let the user decide if the info is acceptable or not.

**Comment by raviwarrier** [comment]
When individually looking up chapters, because I don't know (or don't care enough) I just accept any chapter info that ABS shows without trying to correct them. For people like me, "accept all" (recommendations/suggestions) would be a boon. :)
